styles/components/_filter-toggle.scss

```diff
@@ -0,0 +1,34 @@
+.cmp-filter-toggle {
+  margin-right: space("sm");
+
+  &__input {
+    @include visually-hidden;
+
+    &:checked + span {
+      background-color: color("white");
+      color: color("black");
+    }
+
+    &:focus + span {
+      outline: 2px solid white;
+      outline-offset: 2px;
+    }
+  }
+
+  &__text {
+    border: 1px solid color("white");
+    display: inline-block;
+    font-size: font-size("xs");
+    height: 3.125rem;
+    line-height: 3.125;
+    text-align: center;
+    margin-bottom: space("sm");
+    transition: background-color 250ms, color 250ms ease-in-out;
+    width: 6rem;
+
+    &:hover {
+      outline: 2px solid white;
+      outline-offset: 2px;
+    }
```

Comment:
As we discussed, let's ditch the outline for the hover state and replace it with a background color change. There'll be a refinement card at some point to touch up the details.